ion/src/main/java/com/fasterxml/jackson/dataformat/ion/IonFactory.java

```diff
@@ -462,7 +463,11 @@ protected IonGenerator _createGenerator(OutputStream out, JsonEncoding enc, bool
         // Binary writers are simpler: no alternate encodings
         if (createBinaryWriters()) {
             ctxt.setEncoding(enc);
-            ion = _system.newBinaryWriter(out);
+            if (IonGenerator.Feature.ION_VERSION_1_1.enabledIn(_ionGeneratorFeatures)) {
+                ion = IonEncodingVersion.ION_1_1.binaryWriterBuilder().build(out);
```

Comment:
I'm guessing the answer is "no", but does the `IonObjectMapper` allow you to provide your own `IonWriterBuilder` either when configuring the mapper or writing a value?

Reply:
You can achieve this in a few ways:
1. You can create an IonObjectMapper using an IonFactory. You can provide an IonSystem to an IonFactory. You can provide text or binary writer builders to an IonSystem.
2. You can provide an IonWriter instance to an IonGenerator or to `IonObjectMapper.writeValue`.